README.md

```diff
@@ -1,3 +1,34 @@
+![Tests linux](https://github.com/red-panda-productions/ipc-lib/actions/workflows/testing-linux.yml/badge.svg)
+![Coverage linux](https://github.com/red-panda-productions/ipc-lib/actions/workflows/code-coverage-linux.yml/badge.svg) 
+![Tests windows](https://github.com/red-panda-productions/ipc-lib/actions/workflows/testing-windows.yml/badge.svg)
+![Coverage windows](https://github.com/red-panda-productions/ipc-lib/actions/workflows/code-coverage-windows.yml/badge.svg) 
+# SimulatedDrivingAssistanceSystem
+
 # IPCLib
 
-An open source socket library. IPCLib can be used to create connections between two processes either on the same system or on two different systems, but was mainly created to provide a connection between two processes on the same system for inter process communication. IPCLib provides a server and a client class that can be used to create this behaviour and to easily send data between the two processes. Currently IPCLib only works on Windows, but it will be updated to also work on linux.
+An open source cross platform C++ socket library. 
```

Comment:
"open source" is used as an adjective here, so should be spelled "open-source"
"cross platform" is used as an adjective here, so should be spelled "cross-platform"